### ISSUE TITLE
A bunch of takeaways aren't showing up

### ISSUE THREAD

**Issue by pwim** [tokyodev-inc/tokyodev-devographics#324]
Like for total compensation and role. I guess when keys were renamed these weren't handled properly?

**Comment by SachaG** [comment]
Makes sense, since the takeaways also have the section as part of their key. I'll add aliases (although that's another place where removing the section altogether might make sense).
